emscripten.py

```diff
@@ -1412,7 +1419,9 @@ def setup_function_pointers(function_table_sigs):
 
 
 def create_basic_funcs(function_table_sigs, invoke_function_names):
-  basic_funcs = ['abort', 'assert', 'setTempRet0', 'getTempRet0']
+  basic_funcs = shared.Settings.RUNTIME_FUNCS_TO_IMPORT
+  if shared.Settings.MINIMAL_RUNTIME and not shared.Settings.ASSERTIONS and 'assert' in basic_funcs:
+    basic_funcs.remove('assert') # In MINIMAL_RUNTIME, assert() function is only present in ASSERTIONS-enabled builds.
```

Comment:
Did a quick test removing that `assert` altogether, and ran asm0-3 and binaryen0-3 suites, and got 

```
ERROR: test_async (test_core.binaryen0)
ERROR: test_async_emterpretify (test_core.binaryen0)
ERROR: test_async_emterpretify (test_core.binaryen1)
ERROR: test_async_emterpretify (test_core.binaryen2)
ERROR: test_coroutine_emterpretify_async (test_core.binaryen2)
ERROR: test_coroutine_emterpretify_async (test_core.binaryen3)
ERROR: test_emterpretify (test_core.binaryen0)
```

as failures. Perhaps it can be easy to drop that altogether, but I don't want to go investigating there for now. Adjusted to unconditionally drop `assert` from being imported in MINIMAL_RUNTIME for now.

Reply:
It sounds like emterpretify is using assert inside asm.js - we should just replace that with abort. I'll fix that later today.